docs/src/misc.rst

```diff
@@ -588,7 +611,7 @@ API
 
 .. c:function:: uint64_t uv_hrtime(void)
 
-    Returns the current high-resolution real time. This is expressed in
+    Returns the current high-resolution timestamp. This is expressed in
```

Comment:
Replaced with "timestamp" because "real time" is needlessly confusing when talking about monotonic clocks.